storage/tokudb/CMakeLists.txt

```diff
@@ -82,6 +82,13 @@ endmacro(append_cflags_if_supported)
 set_cflags_if_supported(-Wno-missing-field-initializers)
 append_cflags_if_supported(-Wno-vla)
 
+# Disable warnings for gcc-9 or higher
+IF(CMAKE_COMPILER_IS_GNUCXX AND
+    (CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL 9.0 OR
+     CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 9.0))
+  append_cflags_if_supported(-Wno-address-of-packed-member)
```

Comment:
Why not `MY_CHECK_CXX_COMPILER_FLAG`?

Reply:
It's shorter than:
```
MY_CHECK_CXX_COMPILER_FLAG("-Wno-address-of-packed-member" HAVE_ADDRESS_OF_PACKED_MEMBER)

IF(HAVE_ADDRESS_OF_PACKED_MEMBER)  
  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-address-of-packed-member")
  SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-address-of-packed-member")
ENDIF()
```

Reply:
I missed that `append_cflags_if_supported` is also conditional. Then why is the compiler version check needed?

Reply:
(my concern here is that if this currently isn't an issue with clang, it will be in the future)

Reply:
Currently we have `-Waddress-of-packed-member` warnings only with `gcc-9`. I didn't want to disable these warnings for `clang` and older `gcc` because these warnings may be useful in the future in another code/places.